py-rse/automate.Rmd

```diff
@@ -713,13 +652,13 @@ Finally,
 the third argument is what to do the substitution in,
 which is our list of books' names.
 
-Let's check our worke by adding another command to the `settings` target:
+Let's check the `RESULTS` variable by adding another command to the `settings` target:
 
 ```make
 settings :
-        @echo COUNT: $(COUNT)
-        @echo DATA: $(DATA)
-        @echo RESULTS: $(RESULTS)
+	@echo COUNT: $(COUNT)
+	@echo DATA: $(DATA)
+	@echo RESULTS: $(RESULTS)
 ```
 
 ```shell
```

Comment:
When you run `make settings` here and `make all` a few lines later not all the books are included in the output?

Reply:
Good catch. Before fixing this, do you think we actually need this many novels? Could we communicate the same lessons with just 4-5? The reason I am asking is that the output becomes rather cluttered with the current 17 novels, which makes it harder to overview what is going on, see example below.

```shell
make
```
```text
bin/countwords.py data/anne_of_green_gables.txt > results/anne_of_green_gables.csv
bin/countwords.py data/common_sense.txt > results/common_sense.csv
bin/countwords.py data/count_of_monte_cristo.txt > results/count_of_monte_cristo.csv
bin/countwords.py data/dracula.txt > results/dracula.csv
bin/countwords.py data/emma.txt > results/emma.csv
bin/countwords.py data/ethan_frome.txt > results/ethan_frome.csv
bin/countwords.py data/frankenstein.txt > results/frankenstein.csv
bin/countwords.py data/jane_eyre.txt > results/jane_eyre.csv
bin/countwords.py data/life_of_frederick_douglass.txt > results/life_of_frederick_douglass.csv
bin/countwords.py data/moby_dick.txt > results/moby_dick.csv
bin/countwords.py data/mysterious_affair_at_styles.txt > results/mysterious_affair_at_styles.csv
bin/countwords.py data/pride_and_prejudice.txt > results/pride_and_prejudice.csv
bin/countwords.py data/sense_and_sensibility.txt > results/sense_and_sensibility.csv
bin/countwords.py data/sherlock_holmes.txt > results/sherlock_holmes.csv
bin/countwords.py data/time_machine.txt > results/time_machine.csv
bin/countwords.py data/treasure_island.txt > results/treasure_island.csv
bin/collate.py results/anne_of_green_gables.csv results/common_sense.csv results/count_of_monte_cristo.csv results/dracula.csv results/emma.csv results/ethan_frome.csv results/frankenstein.csv results/jane_eyre.csv results/life_of_frederick_douglass.csv results/moby_dick.csv results/mysterious_affair_at_styles.csv results/pride_and_prejudice.csv results/sense_and_sensibility.csv results/sherlock_holmes.csv results/time_machine.csv results/treasure_island.csv > results/collated.csv
bin/plotcounts.py results/collated.csv --outfile results/plotcounts.png
alpha: 1.5868639581092592
beta: 1.6301737429284708
```

```shell
make settings
```

```text
COUNT: bin/countwords.py
COLLATE: bin/collate.py
PLOT: bin/plotcounts.py
DATA: data/anne_of_green_gables.txt data/common_sense.txt data/count_of_monte_cristo.txt data/dracula.txt data/emma.txt data/ethan_frome.txt data/frankenstein.txt data/jane_eyre.txt data/life_of_frederick_douglass.txt data/moby_dick.txt data/mysterious_affair_at_styles.txt data/pride_and_prejudice.txt data/sense_and_sensibility.txt data/sherlock_holmes.txt data/time_machine.txt data/treasure_island.txt
RESULTS: results/anne_of_green_gables.csv results/common_sense.csv results/count_of_monte_cristo.csv results/dracula.csv results/emma.csv results/ethan_frome.csv results/frankenstein.csv results/jane_eyre.csv results/life_of_frederick_douglass.csv results/moby_dick.csv results/mysterious_affair_at_styles.csv results/pride_and_prejudice.csv results/sense_and_sensibility.csv results/sherlock_holmes.csv results/time_machine.csv results/treasure_island.csv
```

Reply:
Good point. Cutting it down to just 5 books would probably be fine. It would mainly impact the unix shell chapter, which runs word counts on all the books. That lesson would still work with fewer books, it would just need editing.

Reply:
Great! I also added this as a discussion item to the meeting to make sure it is ok with everyone.

Reply:
I'm going to make a separate PR with all the changes related to the reduction of novels, where I will also update the outputs your commented on here.